local.config.php

```diff
@@ -2,17 +2,12 @@
 return [
     'logger' => [
         'log' => false,
-        'priority' => \Zend\Log\Logger::NOTICE,
+        'priority' => \Laminas\Log\Logger::NOTICE,
     ],
     'http_client' => [
         'sslcapath' => null,
         'sslcafile' => null,
     ],
-    'session' => [
```

Comment:
@nomadicoder Why are we removing this?  This was just added for the security scan earlier this summer.

Reply:
@sensei100 Only the Zend->Laminas change needed to be made.